service/worker/batcher/fx.go

```diff
@@ -15,9 +15,8 @@ import (
 
 const (
 	// BatchWFTypeName is the workflow type
-	BatchWFTypeName         = "temporal-sys-batch-workflow"
-	BatchWFTypeProtobufName = "temporal-sys-batch-workflow-protobuf"
-	NamespaceDivision       = "TemporalBatcher"
+	BatchWFTypeName   = "temporal-sys-batch-workflow-protobuf"
+	NamespaceDivision = "TemporalBatcher"
```

Comment:
This is fine but we will be stuck with the `-protobuf` name. You could consider registering the workflow with both names and removing the `-protobuf` suffix after the next release.